src/Compute/Compute/Generated/Disk/Config/NewAzureRmDiskConfigCommand.cs

```diff
@@ -201,6 +206,15 @@ private void Run()
                 vCreationData.SourceResourceId = this.SourceResourceId;
             }
 
+            if (this.IsParameterBound(c => c.UploadSizeInBytes))
+            {
+                if (vCreationData == null)
+                {
+                    vCreationData = new CreationData();
+                }
+                vCreationData.UploadSizeBytes = this.UploadSizeInBytes;
+            }
+
             if (this.IsParameterBound(c => c.EncryptionSettingsEnabled))
```

Comment:
Based on the [design discussion](https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/364#issuecomment-529668985), there should be logic to handle break change, where is the corresponding code?

Reply:
@hyonholee , there are two things to be done for the breaking change:
- Add following attribute to parameter `DiskSizeGB`
```
[CmdletParameterBreakingChange(nameof(DiskSizeGB), ChangeDescription="'DiskSizeInBytes' instead of 'DiskSizeGB' is going to be used when creating upload disks"]
```
- In creating upload disks scenario, you should assign value of DiskSizeGB to DiskSizeInBytes for back compatibility

Both change could be removed in next major release.